server/webapp/WEB-INF/rails/Gemfile.lock

```diff
@@ -110,19 +80,6 @@ GEM
     rack (2.0.6)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (5.2.1)
-      actioncable (= 5.2.1)
-      actionmailer (= 5.2.1)
-      actionpack (= 5.2.1)
-      actionview (= 5.2.1)
-      activejob (= 5.2.1)
-      activemodel (= 5.2.1)
-      activerecord (= 5.2.1)
-      activestorage (= 5.2.1)
-      activesupport (= 5.2.1)
-      bundler (>= 1.3.0)
-      railties (= 5.2.1)
-      sprockets-rails (>= 2.0.0)
```

Comment:
This is the list of gems that `rails` depended on. We only use the subset that's `require`d in `application.rb` https://github.com/gocd/gocd/blob/814b9f8f98fe90e09ffe277cf4b3e1752ae45c9f/server/webapp/WEB-INF/rails/config/application.rb#L3-L8

`require 'rails'` is actually provided by railtie gem.